packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -23,18 +23,32 @@ const DownloadContentTypeL10nMapping: Record<DownloadContentType, string> = {
   'Firefox account recovery key': 'get-data-trio-title-firefox-recovery-key',
 };
 
+export interface GetDataTrioGleanData {
+  id?:
+    | 'two_step_auth_codes_download'
+    | 'two_step_auth_codes_copy'
+    | 'two_step_auth_codes_print';
+  type?: 'setup' | 'inline setup' | 'replace';
+}
+
 export type GetDataTrioProps = {
   value: string | string[];
   contentType?: DownloadContentType;
   onAction?: (type: 'download' | 'copy' | 'print') => void;
   setTooltipVisible: React.Dispatch<React.SetStateAction<boolean>>;
   email: string;
+  gleanDataAttrs?: {
```

Comment:
Is there a reason `gleanDataAttrs` is optional? Perhaps I'm mistaken, but don't we always want this to be defined?

Reply:
We are using this `DataBlock` component in `FlowRecoveryKeyDownload` too, which currently does not have Glean events defined. I see them in "Phase 2 Events" in the taxonomy sheet though - I'll follow up and see if I should tack them on here or just add a note about making this required when we implement them.